docs/content/docs/deployment/memory/network_mem_tuning.md

```diff
@@ -97,20 +97,17 @@ The actual value of parallelism from which the problem occurs is various from jo
 ## Network buffer lifecycle
  
 Flink has several local buffer pools - one for the output stream and one for each input gate. 
-Each of those pools is limited to at most 
+The upper limit of the size of each buffer pool is called the buffer pool **Target**, which is calculated by the following formula.
```

Comment:
```suggestion
The target size of each buffer pool is calculated by the following formula.
```

Reply:
Ok, thanks for rewriting the doc. Fixed.